web/css/joindin.css

```diff
@@ -624,6 +624,19 @@ a.talk.star {
     color: #777;
 }
 
+.event-attendees img {
+    max-width:100%
+}
+
+.event-attendees .row .col {
+    text-align:center;
+}
+
+.event-attendees .row .col .attendeeName {
+    height:40px;
```

Comment:
This seems an overkill... I tried replacing that with a
```
padding-top: 2px;
padding-bottom: 10px;
```
and its seems better to me